sources/packages/forms/src/form-definitions/sfaa2021-22.json

```diff
@@ -26391,10 +26388,10 @@
           "key": "dependantCareCostsPanel",
           "tags": [],
           "properties": {},
-          "customConditional": "",
+          "customConditional": "show = (data.howWillYouBeAttendingTheProgram === \"Full Time\" \n            || data.hasDependents === \"yes\");",
```

Comment:
I think this condition will display the Dependent Care Cost panel and the first question when the intensity is full time even though the student doesn't have any dependent. We probably want to remove the "full time" condition?

Reply:
@lewischen-aot we need the intensity condition because, the following question which is part of the panel needs to be displayed for FT, even though the student does not have any dependent.

![image](https://github.com/bcgov/SIMS/assets/54600590/145c8ac5-7be8-4ddd-8190-156bf4db6db4)